lib/decision_maker.rb

```diff
@@ -0,0 +1,88 @@
+require_relative '../lib/player'
+require_relative '../lib/board'
+
+class DecisionMaker
+  attr_reader :response_message
+  # possibilities        [0]     [1]      [2]      [3]      [4]    [5]      [6]    [7]
+  WIN_POSSIBILITIES = [[0, 1, 2], [3, 4, 5], [6, 7, 8], [0, 3, 6], [1, 4, 7], [2, 5, 8], [0, 4, 8], [2, 4, 6]].freeze
+
+  def initialize
+    symbol_get
+    board_get
+    @moves = { '1' => '1', '2' => '2', '3' => '3', '4' => '4',
+               '5' => '5', '6' => '6', '7' => '7', '8' => '8', '9' => '9' }
+  end
+
+  def symbol_get
+    @players = []
+    # This creates the selector where the users chooses their symbol
+    symbol_list = ['❌', '🔵']
+    prompt = TTY::Prompt.new
+    symb = prompt.enum_select('Player 1! Select your symbol:', symbol_list)
+    # This stores the player objects in an instance array
+    @players << Player.new(symb)
+    @player_turn = @players[0]
+    @players << Player.new(symbol_list.reject { |x| x == symb }[0])
+    # This returns a response message to be printed
+    puts "Player 1 is #{@players[0]} so Player 2 gets #{@players[1]}"
+  end
+
+  def board_get
+    @board = Board.new
+  end
+
+  # once games starts requests 1st move until no more spots are avlbl on board
+  def move_sequence
+    winner = nil
+    until winner || @moves.empty?
+      puts @board.the_actual_board
```

Comment:
Please note that there should be no `puts`/`get.chomp`/`print` statements outside of the `main.rb` file. All other files should contain strictly logic and no display code. Please see requirement below:
![screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/ruby/OOP/milestone3/only_main_print.png)